dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -756,6 +757,44 @@ def test_categorical_diff(self):
         }
         self.assertDictEqual(expected_diff, profile.diff(profile2))
 
+        # Test diff with psi enabled
+        df_categorical = pd.Series(["y", "y", "y", "y", "n", "n", "n", "maybe"])
+        profile = CategoricalColumn(df_categorical.name)
+        profile.update(df_categorical)
+
+        df_categorical = pd.Series(["y", "maybe", "y", "y", "n", "n", "maybe"])
+        profile2 = CategoricalColumn(df_categorical.name)
+        profile2.update(df_categorical)
+
+        # Calculate expected_psi
+        expected_psi = 0
```

Comment:
What about other test cases for non zero?

Reply:
It is non-zero

Reply:
for perc_A, perc_B in zip(bin_perc, bin_perc_2):
            expected_psi += (perc_B - perc_A) * math.log(perc_B / perc_A)

Reply:
could we make `expected_psi` in the `expected_diff` hardcoded as a value? or would that be tough? I just want the expected to be completely disjoint from any calculations. In this case were using the formula for PSI to check our formula for PSI, sort of...